LICENSE

```diff
@@ -631,8 +631,8 @@ to attach them to the start of each source file to most effectively
 state the exclusion of warranty; and each file should have at least
 the "copyright" line and a pointer to where the full notice is found.
 
-    <one line to give the program's name and a brief idea of what it does.>
-    Copyright (C) <year>  <name of author>
+    NewPipe a free lightweight YouTube frontend for Android
```

Comment:
Changing this seems pretty unnecessary as well, as it's also just a guideline.

Reply:
Gitlab auto fills those fields so i tough that it was required, but I'm not sure about that

Reply:
I think legally it would be okay to make these changes. We're discussing this internally right now.